src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/services/EnvelopeFinaliserServiceTest.java

```diff
@@ -135,6 +143,8 @@ public void finaliseEnvelope_should_not_update_other_envelopes_with_ocr_data() {
         assertThat(unaffectedEnvelope.get().getStatus()).isEqualTo(Status.NOTIFICATION_SENT);
         assertThat(unaffectedEnvelope.get().getScannableItems())
             .allMatch(item -> item.getOcrData() != null && item.getOcrValidationWarnings() == null);
+        assertThat(unaffectedEnvelope.get().getCcdId()).isNull();
+        assertThat(unaffectedEnvelope.get().getEnvelopeCcdAction()).isNull();
```

Comment:
It would be nice to check if values aren't changed even when present. Up to you.